include/chemfiles/Frame.hpp

```diff
@@ -355,20 +355,44 @@ class CHFL_EXPORT Frame final {
     /// replaced with the new value.
     ///
     /// @example{tests/doc/frame/set.cpp}
-    void set(std::string name, Property value);
+    void set(std::string name, Property value) {
+        properties_.set(std::move(name), std::move(value));
```

Comment:
Instead of doing this silently, could we emit a warning or throw an error? That or update the documentation to mention that this could lead to an error being thrown when the property is retrieved. -- Fairly minor though.

Reply:
Or we could use a multimap and store multiple properties with the same name and retrieve only if the value and property match, but I don't like this idea as much as explicitly warning the user that replacing a value may lead to an error being thrown.

Reply:
I am not sure what you mean by:

> that this could lead to an error being thrown when the property is retrieved.

Where would the error comes from ?

Reply:
Sorry, I read a `warning` as throw for some reason.